substrate/client/transaction-pool/src/fork_aware_txpool/dropped_watcher.rs

```diff
@@ -164,15 +148,7 @@ where
 							.iter()
 							.all(|h| !self.stream_map.contains_key(h))
```

Comment:
Can you remind me when this second condition here triggers? If I see it correctly it is about views keeping the tx valid but are not in the stream_map. Meaning they went inactive?

Reply:
Yes, exactly. Theoretically inactive views are no longer kept in the `dropped_listener`:
https://github.com/paritytech/polkadot-sdk/blob/9584dbda9ed5fc91b1837f35f401659239fdaff1/substrate/client/transaction-pool/src/fork_aware_txpool/view_store.rs#L422-L426

So this condition maybe some left-over (but it should not hurt us). I will take a closer look on it :)

Reply:
Actually this was check for silently dropped views. I will open a new PR with some improvements in this area.